diff --git a/node_modules/@rbxts/roact-hooked/out/utils/are-deps-equal.lua b/node_modules/@rbxts/roact-hooked/out/utils/are-deps-equal.lua
index 6e2f576..ef94037 100644
--- a/node_modules/@rbxts/roact-hooked/out/utils/are-deps-equal.lua
+++ b/node_modules/@rbxts/roact-hooked/out/utils/are-deps-equal.lua
@@ -18,10 +18,9 @@ local function areDepsEqual(nextDeps, prevDeps)
 			if not (i < #nextDeps) then
 				break
 			end
-			if nextDeps[i + 1] == prevDeps[i + 1] then
-				continue
+			if nextDeps[i + 1] ~= prevDeps[i + 1] then
+				return false
 			end
-			return false
 		end
 	end
 	return true
